_linter/markdown-linter-style.rb

```diff
@@ -0,0 +1,41 @@
+#  Include all rules by default
+all
+
+#  Rule Modifications
+#  ----------------------------------------------------------------------------
+## MD029 - tells linter to allow ordered lists that will increment by 1.  By 
+## default (w/o modification) the linter doesn't accept lists structured like:
+## 1. List item #1
+## 2. List item #2
+## 3. List item #3
```

Comment:
Ah, I get it. I actually prefer the 

```
1.
1.
1.
```

style because you don't have to manually re-number when you delete items, but I think this might be a controversial position on the team, so I'm fine with leaving this as is for now.